adapters/Mintegral/MintegralAdapter/GADMediationAdapterMintegral.h

```diff
@@ -17,9 +17,15 @@
 
 typedef NS_ENUM(NSInteger, GADMintegralErrorCode) {
     /// Missing server parameters.
-    GADMintegralErrorInvalidServerParameters = 101
+    GADMintegralErrorInvalidServerParameters   = 101,
+    /// The ad request was successful, but no ad was returned.
+    GADMintegralErrorAdNotFill = 102,
```

Comment:
Update to 
`GADMintegralErrorAdNotAvailable`.

Reply:
ok, i have changed.